src/domain/routes/index.types.d.ts

```diff
@@ -0,0 +1,27 @@
+type PayViaRoutesArgs = import("lightning").PayViaRoutesArgs
+type RawRoute = { safe_fee: number } & PayViaRoutesArgs["routes"][number]
+type Route = { roundedUpFee: Satoshis; roundedDownFee: Satoshis }
+
+type CachedRoute = { pubkey: Pubkey; route: RawRoute }
+
+declare const cacheKeySymbol: unique symbol
+type CacheKey = string & { [cacheKeySymbol]: never }
+
+type CachedRouteKeyGenerator = {
+  generate(milliSatsAmounts: MilliSatoshis): CacheKey
+}
+
+interface IRoutesRepository {
+  persist: ({
+    key,
+    routeToCache,
+  }: {
+    key
+    routeToCache: CachedRoute
+    time?: Seconds
```

Comment:
missing in params or is not necessary?

Reply:
I put it as an optional param with a default value in the function definition... not sure if that's ok?

Reply:
Removed: https://github.com/GaloyMoney/galoy/pull/594/commits/1939ad2700f915e8ef53253e7f44a2484fd01d04